mlir/lib/Dialect/SparseTensor/IR/SparseTensorDialect.cpp

```diff
@@ -663,6 +664,9 @@ Attribute SparseTensorEncodingAttr::parse(AsmParser &parser, Type type) {
         explicitVal = result;
       } else if (auto result = llvm::dyn_cast<IntegerAttr>(attr)) {
         explicitVal = result;
+      } else if (auto result =
+                     llvm::dyn_cast<::mlir::complex::NumberAttr>(attr)) {
```

Comment:
this ::mlir .. prefix should not be needed right? [see above]
that way, it also fits on one line ;-)

same below